sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalProcessErrorConsumer.java

```diff
@@ -39,9 +40,9 @@
 
   private static final Logger LOG = Loggers.get(ExternalProcessErrorConsumer.class);
 
-  public final void consumeStdError(Process process) {
+  public final Future<?> consumeStdError(Process process) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1452)